src/main/java/com/avispl/symphony/dal/communicator/aver/ptz/AverPTZCommunicator.java

```diff
@@ -0,0 +1,361 @@
+/*
+ * Copyright (c) 2021 AVI-SPL Inc. All Rights Reserved.
+ */
+package com.avispl.symphony.dal.communicator.aver.ptz;
+
+import static com.avispl.symphony.dal.communicator.aver.ptz.AverPTZUtils.convertIntToByteArray;
+
+import java.io.ByteArrayOutputStream;
+import java.io.IOException;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.Iterator;
+import java.util.List;
+import java.util.Optional;
+
+import com.avispl.symphony.api.dal.control.Controller;
+import com.avispl.symphony.api.dal.dto.control.ControllableProperty;
+import com.avispl.symphony.api.dal.dto.monitor.Statistics;
+import com.avispl.symphony.api.dal.error.CommandFailureException;
+import com.avispl.symphony.api.dal.monitor.Monitorable;
+import com.avispl.symphony.dal.communicator.aver.ptz.enums.ReplyStatus;
+import com.avispl.symphony.dal.communicator.aver.ptz.enums.payload.PayloadType;
+import com.avispl.symphony.dal.communicator.aver.ptz.enums.payload.Prefix;
+import com.avispl.symphony.dal.communicator.aver.ptz.enums.payload.command.CommandType;
+import com.avispl.symphony.dal.communicator.aver.ptz.enums.payload.param.AEMode;
+import com.avispl.symphony.dal.communicator.aver.ptz.enums.payload.param.BacklightStatus;
+import com.avispl.symphony.dal.communicator.aver.ptz.enums.payload.param.FocusMode;
+import com.avispl.symphony.dal.communicator.aver.ptz.enums.payload.param.PowerStatus;
+import com.avispl.symphony.dal.communicator.aver.ptz.enums.payload.param.SlowPanTiltStatus;
+import com.avispl.symphony.dal.communicator.aver.ptz.enums.payload.param.SlowShutterStatus;
+import com.avispl.symphony.dal.communicator.aver.ptz.enums.payload.param.WBMode;
+
+/**
+ * Aver PTZ Camera Adapter
+ * Company: Aver
+ *
+ * Properties are divided into groups:
+ * Zoom,Focus,Exposure,ImageProcess
+ *
+ * Supported features are:
+ *
+ * Controlling:
+ * - Power
+ * - Zoom
+ * - Focus
+ * - Image Settings (AE, WB)
+ * - Presets
+ * - Position
+ *
+ * Monitoring:
+ * - Preset recall and status feedback
+ * - Device Mfg
+ * - Device Model
+ * - Serial Number
+ * - Firmware Version
+ *
+ * @author Harry
+ * @version 1.0
+ * @since 1.0
+ */
+public class AverPTZCommunicator extends UDPCommunicator implements Controller, Monitorable {
+	private String cameraID;
+	private String panSpeed;
+	private String tiltSpeed;
+	private int sequenceNumber = 0;
```

Comment:
Do you plan to use this one? If yes, you can keep it, but we have the same name of sequenceNumber in digestResponse method. I think we should change the arg name sequenceNumber to prevent the bug in feature.

Reply:
cameraID, panSpeed and tiltSpeed will be used in the future, so I will keep it.
I will rename arg name sequenceNumber to avoid bugs.